DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/convolution2d/src/convolution_kernel.hpp

```diff
@@ -176,6 +176,7 @@ struct RGB2Grey {
   void operator()() const {
     // this loop is not necessary in hardware since I will pin the
     // `start` bit high, but it makes the testbench easier.
+    [[intel::initiation_interval(1)]]  //
```

Comment:
```suggestion
    [[intel::initiation_interval(1)]]
```

Reply:
The // is there to prevent clang-format from messing with the attributes. without it, the formatter moves the attribute and while loop to the same line.